src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0,   maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="content">
+    <div class="card-content" data-qa="card">
+      <img 
+        class="card-content__image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+      
+      <h2 class="card-content__produсt">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card-content__code">
+        Product code: 195434
+      </div>
+
+      <div class="card-content__reviews">
+        <div  class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card-content__reviews-number">
+          Reviews: 5
+        </div>
+
+      </div>
+      <div class="card-content__price">
+        <p class="card-content__price-name">
+          Price:
+        </p>
+        <p class="card-content__price-cost">
+          $2,199
+        </p>
+      </div>
+
+      <button class="card-content__button">
+        <a
+          href="#"
+          class="card-content__button-link"
+          data-qa="hover"
+        >
+          buy
+        </a>
+      </button>
```

Comment:
it's a redundant button wrapper
```suggestion
        <a
          href="#"
          class="card-content__button-link"
          data-qa="hover"
        >
          buy
        </a>
```